server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ScheduleController.java

```diff
@@ -61,99 +66,145 @@ public class ScheduleController {
-        return Responses.single(scheduleService.rollbackTask(scheduleId, taskId));
+    @RequestMapping(value = "/schedules/{id:[\\d]+}", method = RequestMethod.PUT)
+    public SuccessResponse<Boolean> updateSchedule(@PathVariable Long id, @RequestBody UpdateScheduleReq req) {
+        throw new UnsupportedException();
+
     }
 
+    @RequestMapping(value = "/schedules", method = RequestMethod.POST)
+    public SuccessResponse<Boolean> createSchedule(@RequestBody CreateScheduleReq req) {
+        throw new UnsupportedException();
 
-    @ApiOperation(value = "GetScheduleTaskLog", notes = "获取计划任务日志")
-    @RequestMapping(value = "/schedules/{scheduleId:[\\d]+}/tasks/{taskId:[\\d]+}/log", method = RequestMethod.GET)
-    public SuccessResponse<String> getScheduleTaskLog(@PathVariable Long scheduleId, @PathVariable Long taskId,
-            @RequestParam OdcTaskLogLevel logType) {
-        return Responses.single(scheduleService.getLog(scheduleId, taskId, logType));
+    }
+
+
+    @RequestMapping("/schedules")
+    public PaginatedResponse<ScheduleListResp> list(
+            @PageableDefault(size = Integer.MAX_VALUE, sort = {"id"}, direction = Direction.DESC) Pageable pageable,
+            @RequestParam(required = false, name = "connectionId") List<Long> connectionIds,
+            @RequestParam(required = false, name = "databaseName") String databaseName,
+            @RequestParam(required = false, name = "tenantId") String tenantId,
+            @RequestParam(required = false, name = "clusterId") String clusterId,
+            @RequestParam(required = false, name = "id") Long id,
+            @RequestParam(required = false, name = "status") List<ScheduleStatus> status,
+            @RequestParam(required = false, name = "type") ScheduleType type,
+            @RequestParam(required = false, name = "startTime") Date startTime,
+            @RequestParam(required = false, name = "endTime") Date endTime,
+            @RequestParam(required = false, name = "creator") String creator,
+            @RequestParam(required = false, name = "projectId") Long projectId) {
+
+        QueryScheduleParams req = QueryScheduleParams.builder()
+                .id(id)
+                .connectionIds(connectionIds)
+                .databaseName(databaseName)
+                .tenantId(tenantId)
+                .clusterId(clusterId)
+                .statuses(status)
+                .type(type)
+                .startTime(startTime)
+                .endTime(endTime)
+                .creator(creator)
+                .projectId(projectId)
+                .build();
+        throw new UnsupportedException();
+
+    }
+
+    @RequestMapping(value = "/schedules/{id:[\\d]+}", method = RequestMethod.GET)
+    public SuccessResponse<ScheduleDetailResp> detailSchedule(@PathVariable Long id) {
+        throw new UnsupportedException();
+    }
+
+    @RequestMapping(value = "/schedules/{id:[\\d]+}/dlmRateLimitConfiguration", method = RequestMethod.PUT)
```

Comment:
Is `dlmRateLimitConfiguration` considered as a resource? If not, the URL may be like `/schedules/{id:[\\d]+}/updateDlmRateLimitConfiguration POST`?

Reply:
This is an existing interface with no changes